src/app/visual/fonts/fonts.component.html

```diff
@@ -0,0 +1,13 @@
+<p>
+  Fonts: {{ status }}
+</p>
+
+<ul>
+  <li><span class="icon login"></span> Login</li>
+  <li><span class="icon tps"></span> TPS Reports</li>
+  <li><span class="icon twitter"></span> Twitter</li>
+  <li><i class="fa fa-bank"></i> V4 Bank</li>
+  <li><i class="fas fa-university"></i> V5 FAS - University</li>
+  <li><i class="far fa-chart-bar"></i> V5 FAR - Chart Bar</li>
+  <li><i class="fab fa-amazon-pay"></i> V5 FAB - Amazon Pay</li>
```

Comment:
Hate to be that guy... but can we put each attribute on a new line?
https://host.nxt.blackbaud.com/skyux-team-docs/style#component-html-tags

Reply:
I don't mind changing at all, but can you elaborate?  I only see one attribute, do you mean element? Do you want...

```
  <li>
    <span class="icon login"></span> Custom V4 Login
  </li>
  <li>
    <span class="icon tps"></span> Custom V5 (new icon) TPS Reports
  </li>
...
```